src/renderer/components/+extensions/installed-extensions.tsx

```diff
@@ -10,12 +10,11 @@ import type {
   InstalledExtension,
 } from "../../../extensions/extension-discovery/extension-discovery";
 import { Icon } from "../icon";
-import { List } from "../list/list";
 import { MenuActions, MenuItem } from "../menu";
 import { Spinner } from "../spinner";
 import { cssNames } from "../../utils";
 import { observer } from "mobx-react";
-import type { Row } from "react-table";
+import { ColumnDef, useReactTable, getCoreRowModel, getSortedRowModel } from "@tanstack/react-table";
```

Comment:
Coupling with a 3rd party lib in this arbitrary implementation of "installed extensions" is not ok. We should be using a "Table" or "List" -component which is under our control, and without knowledge of 3rd parties.